rules/game_process.tex

```diff
@@ -127,28 +127,27 @@ \subsubsection{Invalid Goal}
 \subsubsection{Indirect Kick}
 \label{sec:indirect_kick}
 
-The attacking team may only score a goal via indirect kick from any restart into \texttt{playing} from a free kick.
+The attacking team may only score a goal via indirect kick from \texttt{Kick-off}, \texttt{Kick-in} and \texttt{Free Kick}.
 
 \begin{itemize}
   \item A robot may not score a goal from a direct kick, including via deflections.
-  \item The ball must be deliberately played-at a second time (by either another robot, or the same robot) before a goal may be scored.
+  \item The ball must be deliberately played-at a second time by another robot before a goal may be scored.
     A deliberate play at the ball includes successfully kicking the ball, dribbling the ball (and subsequently leaving possession of the ball), or the goalkeeper playing at the ball with its hands.
-  \item If a robot plays the ball to itself, this means that the ball must leave a circular area of at least \qty{0.75}{\metre} around the robot before the ball is played a second time and to be considered as an indirect kick.
+  \item if an indirect free kick is kicked directly into the opponent's goal, a goal kick is awarded.
```

Comment:
_If_ an indirect free kick is kicked directly into the opponent's goal, _the goal does not count and_ a goal kick is awarded.